src/test/java/seedu/address/testutil/TypicalIndexes.java

```diff
@@ -9,4 +9,7 @@
     public static final Index INDEX_FIRST_PERSON = Index.fromOneBased(1);
     public static final Index INDEX_SECOND_PERSON = Index.fromOneBased(2);
     public static final Index INDEX_THIRD_PERSON = Index.fromOneBased(3);
+    public static final Index INDEX_FIRST_DECK = Index.fromOneBased(1);
```

Comment:
Uh these are inside AnakinTypicalIndexes haha

Reply:
Ok, will move these in the next PR